content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -118,30 +124,30 @@ to manage the lifecycle of Istio.
 
 ## Uninstall
 
-* For option 1, uninstall using `kubectl`:
+- For option 1, uninstall using `kubectl`:
```

Comment:
Moving this to an uninstall section within Option 1 means removing the conditional clause. Be sure to include what exactly the reader is uninstalling:
```suggestion
Uninstall the CRD using `kubectl`:
```